src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1811,4 +1826,21 @@ k
             this.canvas.size.height / 2 + this.canvas.panY - km(pos[1])
         ];
     }
+
+    /**
+     * Center a point in the view
+     *
+     * Used only for centering aircraft, this accepts
+     * the x,y of an aircrafts relativePosition
+     *
+     * @for CanvasController
+     * @method _onCenterAircraftInView
+     * @param x {number}    relativePosition.x
+     * @param y {number}    relativePosition.y
+     */
+    _onCenterAircraftInView = ({ x, y }) => {
```

Comment:
should be more like `_centerPointInView()` or something similar to the other method that fires this.